test/filesystemevent/main.d

```diff
@@ -48,135 +48,6 @@ import ocean.task.Task;
 
 import ocean.task.Scheduler;
 
-deprecated class FileModificationTestTask: Task
```

Comment:
You can suppress deprecations via `DFLAGS` when building specific higher level test binary.